js/common/util/seconds_to_duration.js

```diff
@@ -0,0 +1,14 @@
+export default function secondsToDuration(d) {
+  const h = Math.floor(d / 3600);
+  const m = Math.floor((d % 3600) / 60);
+  const s = Math.floor(d % 3600 % 60);
+
+  if (h > 0) {
+    return `${h}:${String(m).padStart(2, 0)}:${String(s).padStart(2, 0)}`;
```

Comment:
[Browser Compatability](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart#Browser_compatibility) for String.prototype.padStart.

![image](https://user-images.githubusercontent.com/1444314/35074945-4beaddb6-fba5-11e7-9ce8-cef80f0d4546.png)

Reply:
Nah, just realized that this is way too aggressive for what we currently have as a min version:

![image](https://user-images.githubusercontent.com/1444314/35081273-0a7fd156-fbc8-11e7-94ab-b86d73e05fc8.png)